js2py/constructors/jsdate.py

```diff
@@ -118,21 +118,29 @@ def utc_strftime(self, pattern):
 
 
 def parse_date(py_string):  # todo support all date string formats
-    try:
+    supported_formats = (
+        "%Y-%m-%dT%H:%M:%S.%fZ",
+        "%Y-%m-%dT%H:%M:%SZ",
+        "%Y-%m-%d",
```

Comment:
Can we also add these 2 formats without Z suffix? 
```
"%Y-%m-%dT%H:%M:%S.%f",
"%Y-%m-%dT%H:%M:%S",
```
This Z suffix requirement is also sometimes confusing the users.

Reply:
@PiotrDabkowski Sure, I'll add those! 👍